mixer/pkg/adapter/test/integration.go

```diff
@@ -118,7 +119,13 @@ type (
 		// Quota is the response from a check call to Mixer
 		Quota map[string]adapter.QuotaResult `json:"Quota"`
 		// Error is the error from call to Mixer
-		Error error `json:"Error"`
+		Error GrpcStatusErr `json:"Error"`
+	}
+
+	// GrpcStatusErr wraps the error returned from Mixer client
+	GrpcStatusErr struct {
```

Comment:
any reason not to just use the Status struct directly?

Reply:
The client returns a status which embeds another type which is actually what needs to be unmarshalled. It's not a problem to use that struct and I can make the change if preferred. I just thought this was less complex.

Reply:
@douglas-reid I've made the request change. Not sure on the failing tests at this point.